lstchain/io/io.py

```diff
@@ -353,14 +353,6 @@ def auto_merge_h5files(
             common_keys=common_keys.difference(keys_to_copy)
 
             with open_file(filename) as file:
-
-                # check value of Table.nrow for keys copied from the first file
-                for k in keys_to_copy:
-                    first_node = merge_file.root[k]
-                    present_node = file.root[k]
-                    if first_node.nrows != present_node.nrows:
-                        raise ValueError("Length of key {} from file {} different than in file {}".format(k, filename, file_list[0]))
-
```

Comment:
I don't understand, this is not specific of the monitoring table... won't this change have unwanted consequences for other tables?

Reply:
`keys_to_copy`currently includes by default:

```
default_keys_to_copy = [
    dl1_params_tel_mon_ped_key, 
    dl1_params_tel_mon_cal_key,
    dl1_params_tel_mon_flat_key,
    dl1_mon_tel_CatB_cal_key,
    dl1_mon_tel_CatB_ped_key,
    dl1_mon_tel_CatB_flat_key
]
```
so all of them are copied from the first subrun and then checked that the length of these tables is the same for all subruns which is not true at least for `dl1_params_tel_mon_flat_key` as reported in #1193

Shall I then remove the first three tables from the `keys_to_copy`? They will be merged then